e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -30,7 +30,7 @@ const { test, expect } = require('../../../../pluginFixtures');
 const { createDomainObjectWithDefaults } = require('../../../../appActions');
 const backgroundImageSelector = '.c-imagery__main-image__background-image';
 const panHotkey = process.platform === 'linux' ? ['Control', 'Alt'] : ['Alt'];
```

Comment:
Has the actual hotkey been changed yet? We should update the `panHotkey` if true